### ISSUE TITLE
if public keys are missing to verify a signature, try to look them up

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-ios#1001]
should be done in parallel when downloading the message. That way user doesn't have to wait (too much) longer. Only for messages that are signed or encrypted.

**Comment by tomholub** [comment]
We've already done a better solution for this as part of #278 / #998 - we refetch / reverify after the message is rendered.
